### DISCUSSION TITLE
Disable all swipe options in the app

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#52054] [Mobile] [answered]
The app has too many swipe left and swipe right options that one cannot disable. Once one accidentally does one of them, he only has a few seconds to push undo. I want to disable them all so I don't have anymore accidents.

**Comment by stevepopovich** [comment]
Hi @jidanni which screens are you referring to here? I would be curious to hear what you think we could change some of our swipe actions to. Thanks for writing in!

**Reply by jidanni** [reply]
Well I think it was notifications, or maybe issues or something. All I know is if you move your finger the wrong way something might get archived or unsubscribe to and if your eyes aren't quick enough you won't realize what's going on and even if you do you have to think about it and then by the time you figured it out it's already gone.

Or maybe it got marked as read or something.

Mostly it's like why can't I find that thing anymore? Oh I see it must be because I moved my fingers some wrong way and now it's disappeared forever. And to get it back one must go  deep down in some menu and dig it out.

Anyway, I bet if Gmail had such options, they would be off by default. Because a lot of older users use gmail. And you can imagine the frustration...

**Reply by stevepopovich** [reply, marked as answer]
Hi @jidanni thanks for the feedback. You can go to Profile -> Settings -> Configure Notifications -> Swipe options. You can't disable them though. We could look into adding that.

**Reply by jidanni** [reply]
Yes, otherwise it's like an automobile, with forward and reverse gears, but no brakes!
